homeassistant/components/onkyo/media_player.py

```diff
@@ -74,23 +60,39 @@
 
 DEFAULT_PLAYABLE_SOURCES = ("fm", "am", "tuner")
 
+SUPPORT_ONKYO = (
+    SUPPORT_VOLUME_SET
+    | SUPPORT_VOLUME_STEP
+    | SUPPORT_VOLUME_MUTE
+    | SUPPORT_TURN_ON
+    | SUPPORT_TURN_OFF
+    | SUPPORT_SELECT_SOURCE
+    | SUPPORT_PLAY_MEDIA
+)
+
+SUPPORT_ONKYO_WO_VOLUME = (
+    SUPPORT_VOLUME_SET
+    | SUPPORT_VOLUME_STEP
+    | SUPPORT_VOLUME_MUTE
+    | SUPPORT_TURN_ON
+    | SUPPORT_TURN_OFF
+    | SUPPORT_SELECT_SOURCE
+    | SUPPORT_PLAY_MEDIA
+)
+
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
-        vol.Optional(CONF_HOST): cv.string,
-        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+        vol.Required(CONF_HOST): cv.string,
+        vol.Optional(CONF_NAME): cv.string,
+        vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
         vol.Optional(CONF_MAX_VOLUME, default=SUPPORTED_MAX_VOLUME): vol.All(
             vol.Coerce(int), vol.Range(min=1, max=100)
```

Comment:
Hey!

I've been testing your new version of the onkyo component, and I've run into some issues with the max_volume setting. As stated on [https://www.home-assistant.io/integrations/onkyo/](https://www.home-assistant.io/integrations/onkyo/), the receiver_max_volume is 200 for newer devices and that is also the case with my receiver. However, with the maximum value delcared here, that is no longer possible. I tested changing it to 200 and this worked (although I just noticed that the max I can set my receiver to is 199)

Reply:
Good catch. I'm not sure a max is even really nessesary...